django_netjsonconfig/base/template.py

```diff
@@ -100,5 +103,21 @@ def clean(self, *args, **kwargs):
         if self.type == 'vpn' and not self.config:
             self.config = self.vpn.auto_client(auto_cert=self.auto_cert)
 
+    def clone(self, user):
+        clone = copy(self)
+        clone.name = self.name + " (CLONE)"
+        clone._state.adding = True
+        clone.pk = None
+        clone.full_clean()
+        clone.save()
+        ct = ContentType.objects.get_for_model(AbstractTemplate)
+        LogEntry.objects.log_action(
+            user_id=user.id,
+            content_type_id=ct.pk,
+            object_id=clone.pk,
+            object_repr=clone.name,
+            action_flag=ADDITION)
```

Comment:
I prefer if the last parenthesis is on its own line:
```python
LogEntry.objects.log_action(
    user_id=user.id,
    content_type_id=ct.pk,
    object_id=clone.pk,
    object_repr=clone.name,
    action_flag=ADDITION
)
```

Reply:
@pawelplsi this isn't solved but was closed, please close after pushing or we will miss it by mistake! :smile:

Reply:
@atb00ker I thought I had solved it already, but now I see I forgot to add this file to my commit. Sorry and thanks for pointing it out :smile: